requirements/base.txt

```diff
@@ -3,3 +3,5 @@ wheel>=0.22
 docopt
 colorlog==2.6.0
 django==1.9.1
+django-mptt==0.8.0
+django-bulk-update==1.1.8
```

Comment:
django-bulk-update isn't used. But in case you need bulk updates, have a look at the options already present in Django.